Website/src/items/renderings/edgewebsite/SpeakersGrid.yml

```diff
@@ -12,10 +15,13 @@ SharedFields:
   Value: Office/16x16/document_tag.png
 - ID: "1a7c85e5-dc0b-490d-9187-bb1dbcb4c72f"
   Hint: Datasource Template
-  Value: /sitecore/templates/Project/edgewebsite/SpeakersGrid
+  Value: 
```

Comment:
Should this be /sitecore/templates/Project/edgewebsite/cmp entities/Edge Speaker

Reply:
no. its the parent folder (bucket) of all the speakers